aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -217,5 +217,4 @@ storage:
 passwd:
   users:
     - name: core
-      ssh_authorized_keys:
-        - "${ssh_authorized_key}"
+      ssh_authorized_keys: ${ssh_keys}
```

Comment:
Have we verified that interpolation works fine here both when specifying one key and when specifying multiple keys? I am concerned about the HCL->YAML conversion.

Reply:
I don't remember any test with multiple keys so far. I'll do it.
I'll try to test in another platform, probably azure.

Reply:
I think we should test *all* platforms... Otherwise we may be breaking something on one platform.

Reply:
I expect it should work: the `ssh_keys` HCL variable is now a json string and json strings are valid YAML. But I have not tested :)

Reply:
I expect it to work, too :-) But faith doesn't help improve software quality ;-)

Reply:
I agree that the only way to be sure it's working would be to test on all platforms, but it's also true that the change is not platform dependent, if is working on packet so far it should work also on other platforms as well.

Reply:
Yes. It's up to you. I've approved the PR anyway :-)

Reply:
For the record: Tested with multiple keys in GCP and it works.